From 7a3981b9a7cc455532edf07059d94320565e5c54 Mon Sep 17 00:00:00 2001
From: Samuel Pitoiset <samuel.pitoiset@gmail.com>
Date: Tue, 10 Oct 2023 16:17:43 +0200
Subject: [PATCH 198/834] radv: enable radv_disable_aniso_single_level=true for
 Zink too

This seems the default OpenGL behavior as well as DX.

Cc: mesa-stable
Signed-off-by: Samuel Pitoiset <samuel.pitoiset@gmail.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/25642>
---
 src/util/00-radv-defaults.conf | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/util/00-radv-defaults.conf b/src/util/00-radv-defaults.conf
index f928bf7dcc5..df89a791df0 100644
--- a/src/util/00-radv-defaults.conf
+++ b/src/util/00-radv-defaults.conf
@@ -46,6 +46,10 @@ Application bugs worked around in this file:
             <option name="radv_disable_aniso_single_level" value="true" />
         </engine>
 
+        <engine engine_name_match="mesa zink">
+            <option name="radv_disable_aniso_single_level" value="true" />
+        </engine>
+
         <engine engine_name_match="Quantic Dream Engine">
             <option name="radv_zero_vram" value="true" />
             <option name="radv_lower_discard_to_demote" value="true" />
-- 
2.42.0

